dist/formats/email_alert_signup/publisher/schema.json

```diff
@@ -139,6 +139,13 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "parent": {
+          "type": "array",
+          "maxItems": 1,
```

Comment:
Not that I've seen thus far. Also seems a little counter-intuitive. I've just realised I can use the pre-existing `guid` or `guid_list` definitions here instead. I could use the latter and call this property `parents` (plural), but can't help but feel this is a case of YAGNI.